### ISSUE TITLE
Can’t remove a single item from picked color history

### ISSUE THREAD

**Issue by fitojb** [stuartlangridge/ColourPicker#65]
You can only clear the whole history.

**Comment by stuartlangridge** [comment]
Yup. I don't think that being able to remove one colour is an important enough use case to warrant cluttering the whole UI with delete buttons, and Pick's goal is to be easy rather than to be comprehensive.

**Comment by fitojb** [comment]
I don’t know about “clutter” when an “×” button could appear on hover next to the (much bigger) “Copy” button.
